wordpress-mobile/WordPress-Android
# Issue/topics you may like

Parent Issue: #12028

## Solution

This PR adds the "topics you may like" section to the `Discover` fragment of the Reader Improvements Phase II. 

## Testing
1. Set the `isReaderImprovementsPhase2Enabled` to true. 
2. Go to the Reader tab. 
3. Click the Discover tab. 
4. You will see the topics you may like at the top of the list. 
5. You can click on one of them to get more details in the `ReaderPostListFragment`. 

## Unit Tests
These will be attached as another PR.

Screenshot | Demo 
--------|-------
 <kbd><img src="https://user-images.githubusercontent.com/1509205/89216567-3cc87b00-d590-11ea-9f2e-b37ea0a6c67c.png" width="320"></kbd>   |       <kbd><img src="https://user-images.githubusercontent.com/1509205/89216573-3df9a800-d590-11ea-9a6b-d05970b3a1f7.gif" width="320"></kbd>

## Reviewing 
Only 1 reviewer is needed but anyone can review. 

## Submitter Checklist
